domains/certificates/Query.API/API/MeasurementsSyncer/MeasurementSyncPublisher.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using MassTransit;
+using Measurements.V1;
+using Meteringpoint.V1;
+using Microsoft.Extensions.Logging;
+
+namespace API.MeasurementsSyncer;
+
+public interface IMeasurementSyncPublisher
+{
+    Task PublishIntegrationEvents(MeteringPoint meteringPoint, MeteringPointSyncInfo syncInfo, List<Measurement> measurements, CancellationToken cancellationToken);
+}
+
+public class MeasurementSyncPublisher : IMeasurementSyncPublisher
+{
+    private readonly EnergyMeasuredIntegrationEventMapper _mapper;
```

Comment:
Do we need a mapper class for this? Seems like mappings other places is on the one or other class? Or extension method? Or maybe this is the way? Lets align as team.

Reply:
Don't we have the integrationEvents library? We could consolidate things there?